TestLocalVariables.py fails on Windows

|  |  |
| --- | --- |
| Bugzilla Link | [43752](https://llvm.org/bz43752) |
| Version | unspecified |
| OS | Windows NT |
| CC | @JDevlieghere,@stella.stamenova,@Teemperor |

## Extended Description
```
FAIL: lldb-api :: lang/c/local_variables/TestLocalVariables.py (475 of 1803)
******************** TEST 'lldb-api :: lang/c/local_variables/TestLocalVariables.py' FAILED ********************
lldb version 10.0.0 (http://llvm.org/svn/llvm-project/lldb/trunk revision 375454)
  clang revision 375454
  llvm revision 375454
LLDB library dir: E:\build_slave\lldb-x64-windows-ninja\build\bin

LLDB import library dir: E:\build_slave\lldb-x64-windows-ninja\build\bin\..\lib

The 'lldb-vscode' executable cannot be located.  The lldb-vscode tests can not be run as a result.

Libc++ tests will not be run because: Don't know how to build with libc++ on windows

libstdcxx tests will not be run because: Don't know how to build with libstdcxx on windows

Skipping following debug info categories: ['dwo', 'dsym', 'gmodules']



Session logs for test failures/errors/unexpected successes will go into directory 'E:/build_slave/lldb-x64-windows-ninja/build/lldb-test-traces'

Command invoked: E:/build_slave/lldb-x64-windows-ninja/llvm/tools/lldb/test/API/dotest.py --arch=x86_64 -s E:/build_slave/lldb-x64-windows-ninja/build/lldb-test-traces -S nm -u CXXFLAGS -u CFLAGS --executable E:/build_slave/lldb-x64-windows-ninja/build/./bin/lldb.exe --dsymutil E:/build_slave/lldb-x64-windows-ninja/build/./bin/dsymutil.exe --filecheck E:/build_slave/lldb-x64-windows-ninja/build/./bin/FileCheck.exe -C E:/build_slave/lldb-x64-windows-ninja/build/./bin/clang.exe --rerun-all-issues --env LLVM_LIBS_DIR=E:/build_slave/lldb-x64-windows-ninja/build/./lib --build-dir E:/build_slave/lldb-x64-windows-ninja/build/tools/lldb/lldb-test-build.noindex --lldb-module-cache-dir E:/build_slave/lldb-x64-windows-ninja/build/tools/lldb/lldb-test-build.noindex/module-cache-lldb\lldb-api --clang-module-cache-dir E:/build_slave/lldb-x64-windows-ninja/build/tools/lldb/lldb-test-build.noindex/module-cache-clang\lldb-api E:\build_slave\lldb-x64-windows-ninja\llvm\tools\lldb\packages\Python\lldbsuite\test\lang\c\local_variables -p TestLocalVariables.py

UNSUPPORTED: LLDB (E:\build_slave\lldb-x64-windows-ninja\build\bin\clang.exe-x86_64) :: test_c_local_variables_dsym (TestLocalVariables.LocalVariablesTestCase) (test case does not fall in any category of interest for this run) 

FAIL: LLDB (E:\build_slave\lldb-x64-windows-ninja\build\bin\clang.exe-x86_64) :: test_c_local_variables_dwarf (TestLocalVariables.LocalVariablesTestCase)

UNSUPPORTED: LLDB (E:\build_slave\lldb-x64-windows-ninja\build\bin\clang.exe-x86_64) :: test_c_local_variables_dwo (TestLocalVariables.LocalVariablesTestCase) (test case does not fall in any category of interest for this run) 

UNSUPPORTED: LLDB (E:\build_slave\lldb-x64-windows-ninja\build\bin\clang.exe-x86_64) :: test_c_local_variables_gmodules (TestLocalVariables.LocalVariablesTestCase) (test case does not fall in any category of interest for this run) 

======================================================================

FAIL: test_c_local_variables_dwarf (TestLocalVariables.LocalVariablesTestCase)

   Test local variable value.

----------------------------------------------------------------------

Traceback (most recent call last):

  File "E:\build_slave\lldb-x64-windows-ninja\llvm\tools\lldb\packages\Python\lldbsuite\test\lldbtest.py", line 1748, in test_method

    return attrvalue(self)

  File "E:\build_slave\lldb-x64-windows-ninja\llvm\tools\lldb\packages\Python\lldbsuite\test\lang\c\local_variables\TestLocalVariables.py", line 55, in test_c_local_variables

    substrs=['(unsigned int) i = 10'])

  File "E:\build_slave\lldb-x64-windows-ninja\llvm\tools\lldb\packages\Python\lldbsuite\test\lldbtest.py", line 2374, in expect

    msg if msg else EXP_MSG(str, output, exe))

AssertionError: False is not True : Variable(s) displayed correctly

Config=x86_64-E:\build_slave\lldb-x64-windows-ninja\build\bin\clang.exe

----------------------------------------------------------------------

Ran 4 tests in 2.094s
```

--- a/lldb/test/API/lang/c/local_variables/TestLocalVariables.py
+++ b/lldb/test/API/lang/c/local_variables/TestLocalVariables.py
@@ -19,7 +19,6 @@ class LocalVariablesTestCase(TestBase):
         self.source = "main.c"
         self.line = line_number(self.source, "// Set break point at this line.")
 
-    @skipIfWindows
     def test_c_local_variables(self):
         """Test local variable value."""
         self.build()

